src/PollingBlockTracker.ts

```diff
@@ -54,6 +52,10 @@ export class PollingBlockTracker
 
   private readonly _setSkipCacheFlag: boolean;
 
+  readonly #onLatestBlockInternalListeners: ((
```

Comment:
Other class variables use `private readonly _`, but I suppose we'll want to change them sometime in the future.

I'm ok with aligning this with others for now if that looks better